### ISSUE TITLE
Dynamic testing

### ISSUE THREAD

**Issue by szsascha** [worksofbarry/NetRPG#19]
Currently, all tests must be defined in Testing.cs. This can be solved more easily.

It could also be a proper test framework, such as XUnit be used.

I think there are a few more things to be done before the implementation. For example, the restructuring of the project structure. Tests should be in a separate folder.

**Comment by worksofliam** [comment]
I agree the testing needs to change. I think that unit testing is a bit easier to implement within Visual Studio because it has bits to generate unit tests for you. I have been developing with vscode so I am not aware of the full steps right now.

This is something I can add to my to-do list :)

**Comment by szsascha** [comment]
What are your plans for #20?

Then we can call RPG programs really simply from C#. This would be really useful for this.
